client/index.html

```diff
@@ -468,6 +468,53 @@ <h1 class="title">Help</h1>
 							</div>
 						</div>
 
+						<h2>Autocompletion</h2>
+
+						<p>
+							Pressing <kbd>Tab</kbd> will autocomplete nicknames, channels, and <kbd>/</kbd>-commands.
+							Users with previous IRC experience will find this familiar; most clients implement tab-completion.
+						</p>
+						<p>
+							You can also start typing one of the following to get a menu of suggestions.
```

Comment:
Right, so this is why I find this a bit confusing: to complete on <kbd>Tab</kbd> press (again, putting aside nicks which has its own logic), you need to start typing one of the mentioned characters which triggers the completion.

I get why you're saying that I think, because _technically_ pressing <kbd>Tab</kbd> has its logic/use case besides dropdown+<kbd>Enter</kbd>, but from a UX standpoint, these are the same thing.
I would structure the doc like this, let me know if it makes sense (of course, ignore my horrible English/wording):

- To open the autocomplete suggestions, type one of the following trigger characters
- To send the highlighted item, press Enter or Tab
- Note: Nicks can also be completed by hitting Tab after a sequence of characters
- Note: You can turn off the autocompletion menu in the settings (I would remove "if unwanted"). Nicks completion when Tab will remain effective.

(By the way, did you mean to add `characters` after `following`?)

Reply:
As mentioned in another review thread, I personally have the menu strategy disabled. I'll be playing with it shortly on a local test instance to see if I can get better wording added to this PR.